lib/ansible/modules/utilities/logic/import_role.py

```diff
@@ -48,6 +48,11 @@
       - Overrides the role's metadata setting to allow using a role more than once with the same parameters.
```

Comment:
What is the purpose of this change? Are they required for testing Online module? Could they be reviewed/merged in a different PR?

Reply:
Sorry, this is messed up from a rebase, I think.

I think I will close this PR and open a new one when I have worked again on this module. It needs better integration on our side before trying to submit it again.

FYI, we are also working on a C14 module which is pretty good at the moment. I think I will try to submit the C14 first. I'll put you in the process review as well